tokio-async-await tests don't compile

## Version
tokio-async-await 0.1.2

## Description

I run `cargo test` in the `tokio-async-await` directory.
The code in the master fails to compile:
```rust
error: unused attribute
  --> src\lib.rs:79:1
   |
79 | #[macro_export]
   | ^^^^^^^^^^^^^^^
   |
note: lint level defined here
  --> src\lib.rs:5:24
   |
5  | #![cfg_attr(test, deny(warnings))]
   |                        ^^^^^^^^
   = note: #[deny(unused_attributes)] implied by #[deny(warnings)]
```

No problem.
There is 1 more issue. Since #585 `tokio` is not referred as `tokio_main` so I also get that error:
`error[E0465]: multiple rlib candidates for tokio found`.
It happens because of the way cargo/rustc looks for the dependencies, and it has 2 rlib files with the same `libtokio` prefix.
Just making sure, is there a reason not to rename it back (to `tokio_main`)?

--- a/tokio-async-await/Cargo.toml
+++ b/tokio-async-await/Cargo.toml
@@ -28,7 +28,7 @@ name = "tokio"
 
 [dependencies]
 futures = "0.1.23"
-tokio = { version = "0.1.8", path = ".." }
+tokio_main = { package = "tokio", version = "0.1.8", path = ".." }
 tokio-io = { version = "0.1.7", path = "../tokio-io" }
 tokio-channel = { version = "0.1.0", path = "../tokio-channel", features = ["async-await-preview"] }
 tokio-reactor = { version = "0.1.5", path = "../tokio-reactor", features = ["async-await-preview"] }

--- a/tokio-async-await/src/lib.rs
+++ b/tokio-async-await/src/lib.rs
@@ -11,7 +11,7 @@ extern crate futures_core;
 extern crate futures_util;
 
 // Re-export all of Tokio
-pub use tokio::{
+pub use tokio_main::{
     // Modules
     clock,
     codec,
@@ -53,7 +53,7 @@ pub mod prelude {
     //!
     //! The prelude may grow over time as additional items see ubiquitous use.
 
-    pub use tokio::prelude::*;
+    pub use tokio_main::prelude::*;
 
     #[doc(inline)]
     pub use crate::async_await::{

